examples/terraform/vmware-cloud-director/README.md

```diff
@@ -20,27 +20,34 @@ Following environment variables or terraform variables can be used to authentica
 | Environment Variable | Terraform Variable |
 |------|---------|
 | VCD_USER | vcd.user |
-| VCD_PASSWORD | vcd.user |
+| VCD_PASSWORD | vcd.password |
 | VCD_ORG | vcd.org |
 | VCD_URL | vcd.url |
 
+To use access token instead of username and password:
+
+
+| Environment Variable | Terraform Variable |
+|------|---------|
+| VCD_AUTH_TYPE | vcd.auth_type |
+| VCD_API_TOKEN | vcd.api_token |
```

Comment:
One thing that confuses me is that:
- we call this API token in Terraform and KubeOne
- it's called refresh token in the CSI config

What's the official name out of those two?

Reply:
Okay, that makes sense. Thanks!